app/models/user.rb

```diff
@@ -51,25 +59,34 @@ def activate
     update_columns(activated: true, activated_at: Time.zone.now)
   end
 
-  def send_activation_email
-    UserMailer.account_activation(self).deliver_now
-  end
-
   def create_reset_digest
     self.reset_token = User.new_token
     update_columns(reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now)
   end
 
   def send_password_reset_email
     UserMailer.password_reset(self).deliver_now
+    microposts.by_user(id)
```

Comment:
dòn này để làm gì ?

Reply:
dòng này có tác dụng là để truy vấn và lấy ra toàn bộ bài viết của người dùng có "id" nhất định

Reply:
vấn đề là sao lại thêm dòng này vào đây ? hàm này là để gửi mail mà sao lại đi truy vấn ở đây

Reply:
em thêm nhầm ạ, em check lại sách thì không có